### PR TITLE
Provision Economics and IT-Humsci site

### PR DESCRIPTION
**Pull request by bryanbraun**
# READY FOR REVIEW

# Summary
Includes the code changes to two new multisites: Economics and IT Humsci. Also includes some docs changes that we discovered needed updating throughout this process.

# Need Review By (Date)
11/25

# Urgency
Low

# Steps to Test
1. Check that the two new multisite directories exist and that they follow our naming conventions.
2. Check that drush aliases have been added for the two new sites and that they are correct.
    - You may be able to test both of these things by creating a local database for each of these sites and and running `drush @[newsite].local si config_installer -y`, to see if it installs successfully.
3. Browse to the following URLs to confirm that the pages load with valid SSL certs (they will redirect to the default site for now):
    - https://economics-dev.stanford.edu
    - https://it-humsci-dev.stanford.edu
    - https://economics-stage.stanford.edu
    - https://it-humsci-stage.stanford.edu
    - https://economics-prod.stanford.edu
    - https://it-humsci-prod.stanford.edu
4. Review the docs changes to ensure they make sense.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docs/NewSite.md [hunk 1]
```diff
@@ -9,20 +9,32 @@ and the url appropriately.
...
-    * Keep with the pattern: [sitename]-dev, [sitename]-stage, and [sitename]-prod
+    * Keep with the pattern: [site-name]-dev, [site-name]-stage, and [site-name]-prod
 1. Wait for the DNS Refresh. This occurs every half hour at :05 and :35 past the hour.
 1. Ensure the Vhost points to Acquia by pinging the url. `ping newvhost.stanford.edu`
 
-## Site Directory and settings.
-1. Disable shield on all sites on the dev and test environments. this will help when the LE certs are generated. `blt drupal:module:uninstall shield [environment:dev,test]`
+## Setup Domains and SSL Certs in Acquia
+1. Disable the shield module for all sites on the dev and test environments. This ensures the the SSL certs can be verified when they are issued.
+  * `blt drupal:module:uninstall shield dev`
```
Comment (silverli) [inline docs/NewSite.md]: issue with these bullets and formatting

File: docs/NewSite.md [hunk 2]
```diff
@@ -9,20 +9,32 @@ and the url appropriately.
...
+  * `blt drupal:module:uninstall shield stage`
+1. Add the new domains to each of the Acquia environments (dev, stage, and prod). You can do this through the Acquia UI, or by using `blt humsci:add-domain`. For example:
+   * `blt humsci:add-domain dev [site-name]-dev.stanford.edu`
+   * `blt humsci:add-domain test [site-name]-test.stanford.edu`
+   * `blt humsci:add-domain prod [site-name]-prod.stanford.edu`
+1. Re-issue the SSL certs for each environment. For example:
+   * `blt humsci:letsencrypt:add-domain --domains=[site-name]-dev.stanford.edu -- dev`
+   * `blt humsci:letsencrypt:add-domain --domains=[site-name]-stage.stanford.edu -- test`
+   * `blt humsci:letsencrypt:add-domain --domains=[site-name]-prod.stanford.edu -- prod`
+1. Get the newly built SSL certs for each environment (`humsci:letsencrypt:get-cert [environment]`) and add them to their corresponding [Acquia Dashboards](https://cloud.acquia.com/app/develop/applications/23a85077-2967-41a4-be22-a84c24e0f81a/environments/265865-23a85077-2967-41a4-be22-a84c24e0f81a/ssl) by clicking "Install SSL Certificate" and filling out the details.
```
Comment (silverli) [inline docs/NewSite.md]: add: `lando blt humsci:letsencrypt:get-cert prod`

**Comment by bryanbraun** [comment]
@pookmish, this should be ready to review now.

I haven't [deployed these sites to the Acquia environments](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/NewSite.md#deploy-and-test) yet, because I wanted you to review it first. The rest of the provisioning steps should be done though.

File: docs/NewSite.md [hunk 3]
```diff
@@ -9,20 +9,32 @@ and the url appropriately.
...
+  * `blt drupal:module:uninstall shield stage`
+1. Add the new domains to each of the Acquia environments (dev, stage, and prod). You can do this through the Acquia UI, or by using `blt humsci:add-domain`. For example:
+  * `blt humsci:add-domain dev [site-name]-dev.stanford.edu`
+  * `blt humsci:add-domain test [site-name]-test.stanford.edu`
+  * `blt humsci:add-domain prod [site-name]-prod.stanford.edu`
+1. Re-issue the SSL certs for each environment. For example:
+  * `blt humsci:letsencrypt:add-domain --domains=[site-name]-dev.stanford.edu -- dev`
+  * `blt humsci:letsencrypt:add-domain --domains=[site-name]-stage.stanford.edu -- test`
+  * `blt humsci:letsencrypt:add-domain --domains=[site-name]-prod.stanford.edu -- prod`
+1. Get the newly built SSL certs for each environment (`blt humsci:letsencrypt:get-cert [environment]`) and add them to their corresponding [Acquia Dashboards](https://cloud.acquia.com/app/develop/applications/23a85077-2967-41a4-be22-a84c24e0f81a/environments/265865-23a85077-2967-41a4-be22-a84c24e0f81a/ssl) by clicking "Install SSL Certificate" and filling out the details.
```
Comment (pookmish) [inline docs/NewSite.md]: This is no longer needed with the command `blt humsci:update-cert [env]` command

**Review by pookmish — CHANGES_REQUESTED** [review]
This looks good. just the one part in the docs is outdated.

File: docs/NewSite.md (hunk 3 above)
Comment (bryanbraun) [reply docs/NewSite.md]: Nice! Updated 👍.

**Review by pookmish — APPROVED** [review]

**Pull request merged**